### PR TITLE
feat(france): separate departements functions

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

File: src/departements.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  if (parseInt(dpt) < 10) {
+    return "0" + dpt;
+  }
+  return dpt;
+};
+
+export const hasPetiteCouronne = departements =>
+  ["75", "92", "93", "94"].filter(dpt =>
+    departements.map(d => d.toString()).includes(dpt)
+  ).length === 4;
```
Comment (revolunet) [inline src/departements.js]:
```suggestion
export const hasPetiteCouronne = departements =>
  ["75", "92", "93", "94"].filter(dpt =>
    departements.map(normalizeDpt).includes(dpt)
  ).length === 4;
```

File: src/departements.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const normalizeDpt = dpt => {
+  if (isNaN(dpt)) {
+    return dpt.toUpperCase(); // 2A, 2B
+  }
```
Comment (douglasduteil) [inline src/departements.js]: @revolunet there is a number conversion before this normalize function ?

```js
isNaN(Number(dpt)) // ?
```

File: src/departements.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  if (parseInt(dpt) < 10) {
+    return "0" + dpt;
+  }
+  return dpt;
```
Comment (douglasduteil) [inline src/departements.js]:
```suggestion
  const dtpNumber = Number(dpt)
  if (isNaN(dtpNumber)) {
    return dpt.toUpperCase(); // 2A, 2B
  }
  if (dtpNumber < 10) {
    return "0" + dpt;
  }
  return dtpNumber;
```

What you think @revolunet ?

Comment (revolunet) [reply src/departements.js]: LGTM

**Comment by douglasduteil** [comment]
haha :P

**Pull request merged**